LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/User.java

```diff
@@ -0,0 +1,96 @@
+package com.example.LeaveManagement.entity;
+
+import javax.persistence.*;
+import java.util.Set;
+
+public class User {
+    @Id
+    private String userName;
+    private String userFirstName;
+    private String userLastName;
+    private String userPassword;
+
+    private String userRole;
+
+    private String assignee;
+
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name="emp_leave_id" ,referencedColumnName = "id")
+    private EmployeeLeave employeeLeave;
+
+    public EmployeeLeave getEmployeeLeave() {
+        return employeeLeave;
+    }
+
+    public void setEmployeeLeave(EmployeeLeave employeeLeave) {
+        this.employeeLeave = employeeLeave;
+    }
+
+    public String getAssignee() {
+        return assignee;
+    }
+
+    public void setAssignee(String assignee) {
+        this.assignee = assignee;
+    }
+
+    @ManyToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+    @JoinTable(name = "USER_ROLE",
+            joinColumns = {
+                    @JoinColumn(name = "USER_ID")
+            },
+            inverseJoinColumns = {
+                    @JoinColumn(name = "ROLE_ID")
+            }
+    )
+    private Set<Role> role;
```

Comment:
Maintain the order for variables and methods

Reply:
Addressed the above comment.